python-frontend/src/test/java/org/sonar/python/semantic/v2/SymbolTableBuilderV2Test.java

```diff
@@ -19,21 +19,78 @@
  */
 package org.sonar.python.semantic.v2;
 
-import org.junit.jupiter.api.Assertions;
+import java.util.Set;
+import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.sonar.plugins.python.api.tree.AssignmentStatement;
 import org.sonar.plugins.python.api.tree.FileInput;
 import org.sonar.plugins.python.api.tree.FunctionDef;
 import org.sonar.plugins.python.api.tree.ImportName;
 import org.sonar.plugins.python.api.tree.Name;
 import org.sonar.python.PythonTestUtils;
+import org.sonar.python.tree.TreeUtils;
 
 class SymbolTableBuilderV2Test {
 
 
   @Test
-  void test() {
-    var builder = new SymbolTableBuilderV2();
+  void testSymbolTableGlobalSymbols() {
```

Comment:
`GlobalSymbols` is a misleading wording (I assume you mean `ModuleLevel`) since it could refer to symbols declared with the `global` keyword.

We probably actually need some tests around those global / nonlocal symbols as well to assert they are not part of any local symbol table (this would probably deserve a separate and isolated unit test though).